### PR TITLE
Update localization for new menu items

### LINKED ISSUE #7611: Update Russian translation
### Description

Баг обнаружен на версии 2.1.0.0. Дублирование пуктов меню в настройках
ApplicationFrameHost_itMozfs1oc
.

### Steps To Reproduce

1. Зайти в настройки
2. Перейти на вкладку предпочтения
ApplicationFrameHost_Yy4VSx4GVE

### Expected behavior

Отсутствие английского варианта дублированного пункта

### Files Version

2.1.0.0

### Windows Version

Windows 11 21H2 22000.376

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/7865298/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/7865299/debug_fulltrust.log)

### PR DESCRIPTION
**Pull request by Pavlusha311245**
**Resolved Issues**
- Closes #7611

**Details of Changes**
- Updated localization file ru-RU

**Validation**
- Built and ran the app

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@Pavlusha311245 before we merge this, can you add your agreement to #7625?

**Comment by Pavlusha311245** [comment]
> @Pavlusha311245 before we merge this, can you add your agreement to #7625?

Done

File: src/Files/MultilingualResources/Files.ru-RU.xlf [hunk 1]
```diff
@@ -3223,67 +3235,67 @@ Files не собирает, хранит, распространяет или 
         </trans-unit>
         <trans-unit id="SettingsOpenFoldersWithOneClick" translate="yes" xml:space="preserve">
           <source>Open folders with a single click</source>
-          <target state="new">Open folders with a single click</target>
+          <target state="new">Открывать папки одним щелчком</target>
```
Comment (yaichenbaum) [inline src/Files/MultilingualResources/Files.ru-RU.xlf:3238]: Can you update the state to translated?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: src/Files/MultilingualResources/Files.ru-RU.xlf (hunk 1 above)
Comment (Pavlusha311245) [reply src/Files/MultilingualResources/Files.ru-RU.xlf:3238]: Those. revert file with my commit?

Comment (yaichenbaum) [reply src/Files/MultilingualResources/Files.ru-RU.xlf:3238]: In order for the translation to work, you need to change `new` to `translated`. https://files.community/docs/contributing/translating-files

Comment (Pavlusha311245) [reply src/Files/MultilingualResources/Files.ru-RU.xlf:3238]: Understood, I'm sorry, I didn't notice)

Comment (Pavlusha311245) [reply src/Files/MultilingualResources/Files.ru-RU.xlf]: Uploaded the changes

File: src/Files/MultilingualResources/Files.ru-RU.xlf [hunk 2]
```diff
@@ -1,2976 +1,2988 @@
...
+<?xml version="1.0" encoding="utf-8"?>
 <xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en-US" target-language="ru-RU" original="FILES/STRINGS/EN-US/RESOURCES.RESW" tool-id="MultilingualAppToolkit" product-name="n/a" product-version="n/a" build-num="n/a">
-    <header>
-      <tool tool-id="MultilingualAppToolkit" tool-name="Multilingual App Toolkit" tool-version="4.0.6915.0" tool-company="Microsoft" />
-    </header>
-    <body>
-      <group id="FILES/STRINGS/EN-US/RESOURCES.RESW" datatype="resx">
-        <trans-unit id="PropertiesCreated.Text" translate="yes" xml:space="preserve">
+	<file datatype="xml" source-language="en-US" target-language="ru-RU" original="FILES/STRINGS/EN-US/RESOURCES.RESW" tool-id="MultilingualAppToolkit" product-name="n/a" product-version="n/a" build-num="n/a">
```
Comment (yaichenbaum) [inline src/Files/MultilingualResources/Files.ru-RU.xlf]: It looks like the last commit added extra tab spaces, can you fix the indentation?

Comment (Pavlusha311245) [reply src/Files/MultilingualResources/Files.ru-RU.xlf]: This is what Visual Studio does when you open the code.

Comment (yaichenbaum) [reply src/Files/MultilingualResources/Files.ru-RU.xlf]: The Multilingual Toolkit is very picky about the indentation

Comment (Pavlusha311245) [reply src/Files/MultilingualResources/Files.ru-RU.xlf]: Didn't know about this feature. Can I do it tomorrow?

Comment (yaichenbaum) [reply src/Files/MultilingualResources/Files.ru-RU.xlf]: Take your time, thank you!

**Pull request closed without merging**